### PR TITLE
Adding basic Zoom API interactivity. Bonus fixing trailing slashes.

### LINKED ISSUE #7: Make trailing slash option
Make trailing slashes optional for server-side endpoints

### PR DESCRIPTION
**Pull request by jdurelli**

### CODE REVIEW

File: RecessApplication/settings.py [hunk 1]
```diff
@@ -24,13 +24,19 @@
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = '_4=x29q$v0$kg862o5k7-)i-ty06e#$=45k$pm@m^cui*z@%n7'
 
+# SECURITY WARNING: keep the secret key used in production secret!
```
Comment (jdurelli) [inline RecessApplication/settings.py:27]: Do we have a better way to do this right now?

Comment (jdurelli) [reply RecessApplication/settings.py:27]: I can always regenerate these so for now it might not be a huge matter

**Comment by jdurelli** [comment]
Closes #29 on RecessApplication

**Comment by jdurelli** [comment]
closes jdurelli/RecessApplication#29

File: RecessApplication/settings.py (hunk 1 above)
Comment (jonathanmcdaniel) [reply RecessApplication/settings.py:27]: We should probably have it read local environment variables that are only stored on our computer and not in source control. Or just in a separate file that we add to the .gitignore

**Review by jonathanmcdaniel — APPROVED** [review]

**Review by gateship1 — APPROVED** [review]

**Review by DylanZeller — APPROVED** [review]
Thanks! Nice job.

**Pull request merged**